clojure-ts-mode.el

```diff
@@ -119,6 +119,18 @@ double quotes on the third column."
   :safe #'booleanp
   :package-version '(clojure-ts-mode . "0.2.3"))
 
+(defcustom clojure-ts-auto-redirect t
```

Comment:
Perhaps `clojure-ts-auto-remap` or `clojure-ts-auto-remap-modes` would be slightly better name.

Reply:
I've renamed it to `clojure-ts-auto-remap`, wasn't sure if it should have a `-p` suffix being a boolean? Feel free to change it further (maybe the commands too, `clojure-ts-activate` -> `clojure-ts-activate-remapping`)